src/index.js

```diff
@@ -1,2 +1,172 @@
+/*eslint-disable*/
 import _ from 'lodash';
 import './style.css';
+import myObject from './modules/function';
+
+const whole = document.querySelector('.toDo');
+const textInput = document.querySelector('.addtext');
+const form = document.querySelector('.form1');
+const submit = document.querySelector('.submit');
+const todoCon = document.querySelector('.todo-container');
+
+// class object
+
+const Array1 = [];
+const addToList = (todovalue) => {
+  const todocontainer = document.createElement('div');
+  todocontainer.className = 'todocontainer';
+  todocontainer.innerHTML += `
+    <input type="checkbox" class="check">
+    <span>${todovalue}</span>
+    <i class="fa-solid fa-ellipsis-vertical menu"></i>
+    <i class="fa-solid fa-trash-can delete"></i>
+  `;
+  todoCon.appendChild(todocontainer);
+
+  const checkbox = document.querySelectorAll('.check');
```

Comment:
To keep things simple you could try having all your query selectors at the beginning of the file for easy readability and scoping